edge-oai-pmh/edge-oai-pmh.postman_collection.json

```diff
@@ -670,27 +1641,27 @@
 	],
 	"variable": [
```

Comment:
@craigmcnally , yes - make sense.
I've alredy done all mentioned changes. Please, review and merge, because I haven't permissions to do this.

Thanks,
Slava